README.md

```diff
@@ -690,9 +690,11 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**IVPN**](https://github.com/ivpn/android-app) <sup>**[[F-Droid](https://f-droid.org/app/net.ivpn.client)]**</sup>
 * [**Mullvad**](https://github.com/mullvad/mullvadvpn-app) <sup>**[[F-Droid](https://f-droid.org/app/net.mullvad.mullvadvpn)]**</sup>
 * [**Mysterium VPN**](https://github.com/mysteriumnetwork/mysterium-vpn-mobile) <sup>**[[F-Droid](https://f-droid.org/app/network.mysterium.vpn)]**</sup>
+* [**NekoBox for Android**](https://github.com/MatsuriDayo/NekoBoxForAndroid)
 * [**ProtonVPN**](https://github.com/ProtonVPN/android-app) <sup>**[[F-Droid](https://f-droid.org/app/ch.protonvpn.android)]**</sup>
 * [**Riseup VPN**](https://0xacab.org/leap/bitmask_android) <sup>**[[F-Droid](https://f-droid.org/app/se.leap.riseupvpn)]**</sup>
 * [**Tailscale**](https://github.com/tailscale/tailscale-android) <sup>**[[F-Droid](https://f-droid.org/app/com.tailscale.ipn)]**</sup>
+* [**v2rayNG**](https://github.com/2dust/v2rayNG)
```

Comment:
I'm very sceptical about this one. Izzy has removed it from his repository already (see [this discussion](https://gitlab.com/fdroid/rfp/-/issues/1168)).

Reply:
Sad. One of the biggest clients, after all. But reasons are serious. Will delete it then 😃

Reply:
Thanks, PR is ready to merge then :+1:

Reply:
[Ready to merge!](https://github.com/offa/android-foss/pull/304/commits/7abcfd2124b2060fb3b13b09f178f593c087e193)